packages/cli/src/util/createServer.ts

```diff
@@ -96,13 +91,9 @@ function logCLIMessage(message: string) {
   });
 }
 
-function isHttp(spec: string) {
-  return !!spec.match(/^https?:\/\//);
-}
-
 export type CreatePrismOptions = {
   dynamic: boolean;
   host?: string;
   port: number;
-  spec: string;
+  operations: IHttpOperation[];
```

Comment:
You can see from here that Prism does not care about the source of the operations. You need to supply them somehow.